modules/core/src/main/java/org/apache/synapse/SynapseConstants.java

```diff
@@ -230,6 +230,24 @@
         public static final String STATISTICS_STACK ="synapse.statistics.stack";     
         
         public static final String SYNAPSE_STATISTICS_STATE = "synapse.statistics.state";
+
+        /** key to message context statistic entry */
```

Comment:
Please use proper linguistics when adding comments. For example, in this particular case, use capitals for the first letter of the sentence.